charts/nri-kube-events/values.yaml

```diff
@@ -1,82 +1,31 @@
-#   contains it.
-#
-# - Specify the Kubernetes cluster name.
-#   https://docs.newrelic.com/docs/kubernetes-monitoring-integration
-#
-# licenseKey:
-# customSecretName:
-# customSecretLicenseKey:
-# cluster:
-#
-# IMPORTANT: the previous values can also be set as global so that they
-# can be shared by other newrelic product's charts
-#
-# global:
-#   licenseKey:
-#   customSecretName:
-#   customSecretLicenseKey:
-#   cluster:
-
-agentHTTPTimeout: "30s"
-
-# Kube-events container will spin up a infra-agent(event-forwarder) as a sidecar container to forward
-# events to New Relic One endpoint. However, due to security restrictions, some users might require
-# to use an http proxy to route traffic over the internet. If this is the case for you, set this value
-# to your http proxy endpoint.
-proxy: {}
-
-sinks:
-  # Enable the stdout sink to also see all events in the logs.
-  stdout: false
-  # The newRelicInfra sink sends all events to New relic.
-  newRelicInfra: true
-
+# -- Override the name of the chart
 nameOverride: ""
+# -- Override the full name of the release
+fullnameOverride: ""
 
+# -- Images used by the chart for the integration and agents
+# @default -- See `values.yaml`
 images:
+  # -- Image for the New Relic Kubernetes integration
+  # @default -- See `values.yaml`
   integration:
     registry:
     repository: newrelic/nri-kube-events
-    tag: ""
+    tag: 1.8.0
     pullPolicy: IfNotPresent
+  # -- Image for the New Relic Infrastructure Agent sidecar
+  # @default -- See `values.yaml`
   agent:
     registry:
     repository: newrelic/k8s-events-forwarder
     tag: 1.22.0
     pullPolicy: IfNotPresent
-  ## It is possible to specify docker registry credentials
-  ## See https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # -- The secrets that are needed to pull images from a custom registry.
   pullSecrets: []
   # - name: regsecret
 
-# For unprivileged mode - default is 1000.
-# Can also be modified for OpenShift clusters where the runAsUser range
-# may be restricted.  See OpenShift docs for more information.
-# https://www.openshift.com/blog/a-guide-to-openshift-and-uids
-# runAsUser:
-
-# Custom container security context with default values:
-# - privileged: false
-# - allowPrivilegeEscalation: false
-# - readOnlyRootFilesystem: true
-containerSecurityContext:
-
-# Custom pod security context with default values:
-# - runAsUser: 1000
-# - runAsNonRoot: true
-podSecurityContext:
-
-resources:
-  {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+# -- Resources available for this pod
+resources: {}
```

Comment:
should we add resources by default? (I know it is not the place for this, we might open an issue)

Reply:
As you know I'm a strong advocate for leaving them undefined, as they might vary a lot depending on hardware and cluster size.

Reply:
As Helm says:
> We usually recommend not to specify default resources and to leave this as a conscious choice for the user. This also increases chances charts run on environments with little resources, such as Minikube.

If the chart did not have `resources` set in the values before I touch it, I do not touch them.